src/replica/CMakeLists.txt

```diff
@@ -0,0 +1,500 @@
+    SqlApp.cc
+    SqlApp.h
+    SqlCreateDbJob.cc
+    SqlCreateDbJob.h
+    SqlCreateDbRequest.cc
+    SqlCreateDbRequest.h
+    SqlCreateIndexesJob.cc
+    SqlCreateIndexesJob.h
+    SqlCreateIndexesRequest.cc
+    SqlCreateIndexesRequest.h
+    SqlCreateTableJob.cc
+    SqlCreateTableJob.h
+    SqlCreateTableRequest.cc
+    SqlCreateTableRequest.h
+    SqlCreateTablesJob.cc
+    SqlCreateTablesJob.h
+    SqlCreateTablesRequest.cc
+    SqlCreateTablesRequest.h
+    SqlDeleteDbJob.cc
+    SqlDeleteDbJob.h
+    SqlDeleteDbRequest.cc
+    SqlDeleteDbRequest.h
+    SqlDeleteTableJob.cc
+    SqlDeleteTableJob.h
+    SqlDeleteTablePartitionJob.cc
+    SqlDeleteTablePartitionJob.h
+    SqlDeleteTablePartitionRequest.cc
+    SqlDeleteTablePartitionRequest.h
+    SqlDeleteTableRequest.cc
+    SqlDeleteTableRequest.h
+    SqlDisableDbJob.cc
+    SqlDisableDbJob.h
+    SqlDisableDbRequest.cc
+    SqlDisableDbRequest.h
+    SqlDropIndexesJob.cc
+    SqlDropIndexesJob.h
+    SqlDropIndexesRequest.cc
+    SqlDropIndexesRequest.h
+    SqlEnableDbJob.cc
+    SqlEnableDbJob.h
+    SqlEnableDbRequest.cc
+    SqlEnableDbRequest.h
+    SqlGetIndexesJob.cc
+    SqlGetIndexesJob.h
+    SqlGetIndexesRequest.cc
+    SqlGetIndexesRequest.h
+    SqlGrantAccessJob.cc
+    SqlGrantAccessJob.h
+    SqlGrantAccessRequest.cc
+    SqlGrantAccessRequest.h
+    SqlJob.cc
+    SqlJob.h
+    SqlJobResult.cc
+    SqlJobResult.h
+    SqlQueryJob.cc
+    SqlQueryJob.h
+    SqlQueryRequest.cc
+    SqlQueryRequest.h
+    SqlRemoveTablePartitionsJob.cc
+    SqlRemoveTablePartitionsJob.h
+    SqlRemoveTablePartitionsRequest.cc
+    SqlRemoveTablePartitionsRequest.h
+    SqlRequest.cc
+    SqlRequest.h
+    SqlResultSet.cc
+    SqlResultSet.h
+    SqlSchemaUtils.cc
+    SqlSchemaUtils.h
+    StatusRequest.cc
+    StatusRequest.h
+    StatusRequestBase.cc
+    StatusRequestBase.h
+    StopRequest.cc
+    StopRequest.h
+    StopRequestBase.cc
+    StopRequestBase.h
+    SuccessRateGenerator.cc
+    SuccessRateGenerator.h
+    SyncApp.cc
+    SyncApp.h
+    Task.cc
+    Task.h
+    TestEchoQservMgtRequest.cc
+    TestEchoQservMgtRequest.h
+    TransactionsApp.cc
+    TransactionsApp.h
+    Url.cc
+    Url.h
+    VerifyApp.cc
+    VerifyApp.h
+    VerifyJob.cc
+    VerifyJob.h
+    WorkerApp.cc
+    WorkerApp.h
+    WorkerDeleteRequest.cc
+    WorkerDeleteRequest.h
+    WorkerEchoRequest.cc
+    WorkerEchoRequest.h
+    WorkerFindAllRequest.cc
+    WorkerFindAllRequest.h
+    WorkerFindRequest.cc
+    WorkerFindRequest.h
+    WorkerIndexRequest.cc
+    WorkerIndexRequest.h
+    WorkerProcessor.cc
+    WorkerProcessor.h
+    WorkerProcessorThread.cc
+    WorkerProcessorThread.h
+    WorkerReplicationRequest.cc
+    WorkerReplicationRequest.h
+    WorkerRequest.cc
+    WorkerRequest.h
+    WorkerRequestFactory.cc
+    WorkerRequestFactory.h
+    WorkerServer.cc
+    WorkerServer.h
+    WorkerServerConnection.cc
+    WorkerServerConnection.h
+    WorkerSqlRequest.cc
+    WorkerSqlRequest.h
```

Comment:
I (personally) prefer explicit lists to globbing in this application.  Let's discuss at the next cowork -- if the majority opinion is for globbing, will change on a subsequent ticket.

Reply:
Philosophy question: if it doesn't throw a link error, and the unit and integration tests pass, was it really needed anyway? 🤔

Reply:
From my point of view, I don't see any significant benefits in "globbing". To me, this is just a matter of taste. And when it comes to taste a decision should be up to the developer .. not to the "reviewing committee".
By the way, mentioning files explicitly does have one benefit - an ability to exclude (if needed) some files from the compilation. This may be especially important during the prototyping or active development phase of some new features.